packages/flutter/lib/fix_data.yaml

```diff
@@ -17,6 +17,294 @@
 
 version: 1
 transforms:
+  # Changes made in https://github.com/flutter/flutter/pull/97972/
+  - title: "Migrate 'ThemeData.toggleableActiveColor' to individual themes"
+    date: 2022-05-18
+    element:
+      uris: [ 'material.dart' ]
+      constructor: ''
+      inClass: 'ThemeData'
+    changes:
+    - kind: 'removeParameter'
+      name: 'toggleableActiveColor'
+    - kind: 'addParameter'
+      index: 95
+      name: 'checkboxTheme'
+      style: optional_named
+      argumentValue:
+        expression: "CheckboxThemeData(\n
+                       fillColor: MaterialStateProperty.resolveWith<Color?>((Set<MaterialState> states) {\n
+                         return states.contains(MaterialState.selected) ? {% toggleableActiveColor %} : null;\n
+                       }),\n
+                     ),"
+        requiredIf: "toggleableActiveColor != '' && checkboxTheme == ''"
+    - kind: 'addParameter'
+      index: 96
```

Comment:
Like here for example, above there is an option for adding `checkboxTheme` at index 95, and here this option for `checkboxTheme` is index 96?

Reply:
Added matching indices